php/Dockerfile-7.0

```diff
@@ -24,7 +24,7 @@ RUN apt-get update -q -y \
         libicu52 \
         libxslt1.1 \
         libmemcachedutil2 \
-# Required by eZ Commerce
+# Required by eZ Commerce (added here to keep images in sync, feature wise)
```

Comment:
If such comments (required by eZ commerce) is needed, that should be added to the soap line below to..
IMO it is not needed though ( reason is anyway in gitlog )

Reply:
@vidarl I removed your comments about eZ Commerce from all the Dockerfiles here.
Could you re-review the PR?

Reply:
done